### ISSUE TITLE
[declarations] falsi positivi

### ISSUE THREAD

**Issue by dennisangemi** [opendatasicilia/accessibility-monitoring#6]
https://www.comune.barrea.aq.it/ nè `calinkx` (curl based) nè `balinkx` (browser based) riescono ad individuare la dichiarazione ma c'è! 

```
./scripts/balinkx.sh https://www.comune.barrea.aq.it/
balinkx: No accessibility links found on website: https://www.comune.barrea.aq.it/

./scripts/calinkx.sh https://www.comune.barrea.aq.it/
calinkx: Failed to extract accessibility references: https://www.comune.barrea.aq.it/
```

in particolare, chrome trova questo:

```
google-chrome-stable --headless=new --no-sandbox --disable-gpu \
    --user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 \
    (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36" \
    --log-level=3 \
    --dump-dom "https://www.comune.barrea.aq.it/"

<!DOCTYPE html>
<html xml:lang="it-IT" xmlns="http://www.w3.org/1999/xhtml" lang="it-IT" style="--color-primary: undefined; --color-secondary: undefined; --color-tertiary: undefined; --color-quaternary: undefined;"><head>
        <meta charset="utf-8">
        <link rel="icon" href="/favicon.ico">
        <meta name="viewport" content="width=device-width,initial-scale=1">
        <meta name="theme-color" content="#000000">
        <meta name="description" content="Accedi ai servizi , agli uffici, alle notizie, agli avvisi e alla modulistica con collegamenti all'Amministrazione Trasparente e all'Albo Pretorio.">
        <meta property="og:title" content="Comune di Barrea">
        <meta property="og:description" content="Accedi ai servizi , agli uffici, alle notizie, agli avvisi e alla modulistica con collegamenti all'Amministrazione Trasparente e all'Albo Pretorio.">
        <meta property="og:image" content="https://www.comune.barrea.aq.it/backend/media/news/default.webp">
        <link rel="apple-touch-icon" href="/maskable_icon_x192.png">
        <link rel="manifest" href="/manifest.json">
        <link rel="stylesheet" href="/styles.css">
        <title>Comune di Barrea</title>
        <script type="text/javascript" async="" src="https://analytics.actainfo.cloud/matomo.js"></script><script src="/env.js"></script>
        <script type="text/javascript">var _paq=window._paq=window._paq||[];_paq.push(["disableCookies"]),_paq.push(["trackPageView"]),_paq.push(["enableLinkTracking"]),function(){var a="https://analytics.actainfo.cloud/";_paq.push(["setTrackerUrl",a+"matomo.php"]),_paq.push(["setSiteId","8"]);var e=document,t=e.createElement("script"),p=e.getElementsByTagName("script")[0];t.type="text/javascript",t.async=!0,t.src=a+"matomo.js",p.parentNode.insertBefore(t,p)}()</script>
        <script defer="defer" src="/static/js/main.f35f9f60.js"></script>
        <link href="/static/css/main.85534cd5.css" rel="stylesheet">
        <link href="/custom.css" rel="stylesheet">
    </head>
    <body>
        <noscript>You need to enable JavaScript to run this app.</noscript>
        <div id="root" class="row"><div></div></div>

    <script>"serviceWorker"in navigator&&window.addEventListener("load",(()=>{navigator.serviceWorker.register("./serviceworker.js").then((e=>console.log("Success: ",e.scope))).catch((e=>console.log("Failure: ",e)))}))</script>

</body></html>
```

**Comment by dennisangemi** [comment]
è bastato aggiungere

https://github.com/opendatasicilia/accessibility-monitoring/commit/ca7570331457928849e46d9c336570d85b7886d7#diff-5cb2513f7c0abf277c46db96651f5dc516ab793e4a7b1a4b70ecfff83d9d3382R20

e adesso

```
./scripts/balinkx.sh https://www.comune.barrea.aq.it/
{"href":"https://form.agid.gov.it/view/850b62b0-7a46-11ef-b8f5-d7088035630a","text":"Dichiarazione di accessibilità"}
```
